src/Uno.UWP/Devices/Bluetooth/GenericAttributeProfile/GattDeviceServicesResult.cs

```diff
@@ -1,14 +1,20 @@
-﻿using System;
+﻿#nullable enable
 using System.Collections.Generic;
-using System.Text;
 
 namespace Windows.Devices.Bluetooth.GenericAttributeProfile
 {
 	public partial class GattDeviceServicesResult
 	{
-		public  byte? ProtocolError { get; internal set; }
-		public  IReadOnlyList<GattDeviceService> Services { get; internal set; }
-		public  GattCommunicationStatus Status { get; internal set; }
+		public byte? ProtocolError { get; internal set; }
+		public IReadOnlyList<GattDeviceService> Services { get; internal set; }
+		public GattCommunicationStatus Status { get; internal set; }
+
+		private GattDeviceServicesResult(IReadOnlyList<GattDeviceService> services)
+		{
+			// dummy for Error CS8618  Non-nullable property 'Value' must contain a non-null value when exiting constructor. Consider declaring the property as nullable.
+			Services = services;
```

Comment:
Empty list instead.

Reply:
This constructor probably would be used in 'real code'.
This object cannot be created as such (no public ctor), it can only be returned inside other structs/classes from Bluetooth implementation.